command/sub8_missions/nodes/move_command

```diff
@@ -39,6 +45,34 @@ def main(args):
         fprint("Odom found!", msg_color='green')
 
         action_kwargs = {'speed': float(args.speed), 'blind': bool(args.blind)}
+        if args.gteleport:
+            try:
+                state_set_pub = yield nh.advertise('gazebo/set_model_state', ModelState)
+                x = args.gteleport[0]
+                y = args.gteleport[1]
+                z = args.gteleport[2]
+                modelstate = ModelState(
+                    model_name='sub8',
+                    pose = Pose(position=Point(x,y,z), orientation=Quaternion(0,0,0,0)),
+                    twist = Twist(linear=Point(0,0,0),angular=Point(0,0,0)),
+                    reference_frame= 'world')
+                    #Sometimes you need to sleep in order to get the thing to publish
+                    #Apparently there is a latency when you set a publisher and it needs to actually hook into it.
+                    #Wierd ROS stuff.
+                #print("Before")
```

Comment:
We're trying to get rid of debug / dead code in the repo. If you don't imagine ever having these lines uncommented in "production" code, you should remove them